src/index.html

```diff
@@ -17,11 +17,433 @@
     />
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="styles/main.scss"
+    />
+    <link
+      rel="stylesheet"
+      href="styles/header.scss"
     />
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        href="#"
+        class="logo"
+      >
+        <img
+          src="./images/logo.png"
+          alt="Moyo Logo"
+          class="logo__image"
+        />
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link is-active"
+            >
+              Apple
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              Samsung
+            </a>
+          </li>
+          <li class="nav__item">
```

Comment:
Add the blank line between the links 
```suggestion
          </li>
          
          <li class="nav__item">
```